regions/shapes/polygon.py

```diff
@@ -35,8 +40,10 @@ def area(self):
         raise NotImplementedError
 
     def contains(self, pixcoord):
-        # TODO: needs to be implemented
-        raise NotImplementedError
+        return points_in_polygon(pixcoord.x.astype(float),
```

Comment:
There's this error:
https://travis-ci.org/astropy/regions/jobs/180103544#L819
```
E       AttributeError: 'int' object has no attribute 'astype'
```
Besides fixing the error, may I suggest you change the code to do the type conversions on separate lines before the `points_in_polygon` call and put them in temp variables?
The efficiency is the same, and the error message for invalid data like here will improve.
At the moment from the traceback it's not clear which variable / line is the problematic one.